src/shared/Core.Tests/TokenEndpointResponseJsonTest.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Text.Json;
+using GitCredentialManager.Authentication.OAuth.Json;
+using Xunit;
+
+namespace Core.Tests;
+
+public class TokenEndpointResponseJsonTest
+{
+    [Fact]
+    public void TokenEndpointResponseJson_Deserialize_Uses_Scope()
+    {
+        var scopesString = "x,y,z";
+        var json = $"{{\"access_token\": \"\", \"token_type\": \"\", \"expires_in\": \"1000\", \"scopes\": \"a,b,c\", \"scope\": \"{scopesString}\"}}";
+
+        var result = JsonSerializer.Deserialize<TokenEndpointResponseJson>(json,
+            new JsonSerializerOptions
+            {
+                PropertyNameCaseInsensitive = true,
+            });
+
+        Assert.Equal(scopesString, result.Scope);
```

Comment:
Can we verify the other properties are correctly deserialised too?

Reply:
Good catch. Made the necessary corrections so that all properties are correctly deserialized.